.php-cs-fixer.php70types.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+$config = require '.php-cs-fixer.dist.php';
+
+$config->getFinder()->notPath([
+    // @TODO 4.0 change interface to be fully typehinted and remove the exceptions from this list
```

Comment:
basically i let to change internal/private code, and any changes to public non-internal code I added here as exceptions